lib/context/AuthoritiesSearchContext.js

```diff
@@ -67,12 +73,42 @@ const AuthoritiesSearchContextProvider = ({
     option: searchParams.qindex,
   });
 
+  useEffect(() => {
+    paramsBySegment.current[initialSegment] = {
+      searchInputValue: initialSearchInputValue,
+      searchQuery: initialSearchQuery,
+      searchIndex: initialSearchIndex,
+      searchDropdownValue: initialDropdownValue,
+      filters: getInitialFilters(),
+      advancedSearchRows: [],
+    };
+  }, []);
```

Comment:
Let's add turn off the warning 
`// eslint-disable-next-line react-hooks/exhaustive-deps`